src/dmg/apu.cpp

```diff
@@ -584,10 +584,10 @@ void dmg_audio_callback(void* _apu, u8 *_stream, int _length)
 	//Set correct length for stereo
 	if(config::use_stereo) { length /= 2; }
 
-	s16 channel_1_stream[length];
-	s16 channel_2_stream[length];
-	s16 channel_3_stream[length];
-	s16 channel_4_stream[length];
+	s16* channel_1_stream = new s16[length];
+	s16* channel_2_stream = new s16[length];
+	s16* channel_3_stream = new s16[length];
+	s16* channel_4_stream = new s16[length];
```

Comment:
This part is kinda ugly. I guess MSVC doesn't like switching to C99... Anyway, something like this is preferred:

    std::vector<s16> channel_1_stream(length);
    std::vector<s16> channel_2_stream(length);
    etc...

This avoids manual memory management and gets rid of having to call **delete**